azure-pipelines.yml

```diff
@@ -0,0 +1,81 @@
+name: 3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r) # equal to $(Build.BuildNumber)
+trigger:
+  branches:
+    include:
+    - dev
+    - master
+    - hotfix/*
+    - release/*
+variables:
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_SDK_VERSION: '3.0.100-preview3-010431'
+jobs:
+- job: Linux
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+  - bash: sudo sh .build/setup_postgres.sh
+    displayName: 'Start PostgreSQL'
+  - task: DotNetCoreInstaller@0
+    displayName: 'Install .NET Core SDK'
+    inputs:
+      packageType: 'sdk'
+      version: '$(DOTNET_SDK_VERSION)'
+  - task: DotNetCoreCLI@2
+    displayName: 'Test'
+    inputs:
+      command: 'test'
+      configuration: 'Debug'
+      projects: 'test/**/*.csproj'
+      publishTestResults: true
+- job: Windows
+  pool:
+    vmImage: vs2017-win2016
```

Comment:
VS2019 was released - we should probably switch to that... [The docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#use-a-microsoft-hosted-agent) still say that it's in preview (it was released a couple of days ago), but we should still use image `windows-2019` unless there's a failure...